awx/ui/client/src/network-ui/debug.directive.js

```diff
@@ -3,7 +3,18 @@
 const templateUrl = require('~network-ui/debug.partial.svg');
 
 function debug () {
-  return { restrict: 'A', templateUrl};
+  return {
+      restrict: 'A',
+      templateUrl,
+      link: function(scope){
+          $('.NetworkUI__debug-text').each(function(index, option){
+             let startingY = 15;
+             let offset = 20;
+             let y = startingY + (index * offset);
+             option.setAttribute('y', y);
+          });
+      }
+  };
```

Comment:
@benthomasson I did this to re-align all the debug text. All the gaps were annoying to weed through, and I wasn't able to see the things at the bottom of the debug view. 
![image](https://user-images.githubusercontent.com/7010629/37691794-927c64c8-2c71-11e8-95f2-06f7e0a9c5ce.png)